plugins/seccomp/src/PoliciesTable.tsx

```diff
@@ -0,0 +1,372 @@
+                  }}
+                >
+                  {data.metadata.name}
+                </Link>
+              ),
+            },
+            {
+              label: 'Pod',
+              getter: (data: any) => parseSeccompAnnotations(data, 'pod', namespace),
+            },
+            {
+              label: 'Container',
+              getter: (data: any) => parseSeccompAnnotations(data, 'container', namespace),
+            },
+            {
+              label: 'Date',
+              getter: (data: any) => new Date(data.metadata.creationTimestamp).toLocaleString(),
+            },
+          ]}
+          data={tableData}
+        />
+      </SectionBox>
+    </Box>
+  );
+}
+
+function SeccompPoliciesTableView(props: any) {
+  const { seccompPolicies, appliedPolicies, danglingPolicies } = props;
+  const [isEditorOpen, setIsEditorOpen] = React.useState(false);
+  const [editorValue, setEditorValue] = React.useState(null);
+  const [errorMessage, setErrorMessage] = React.useState(null);
+  const { namespace } = useParams();
+  const [massagedSeccompPolicies, setMassagedSeccompPolicies] = React.useState(seccompPolicies);
+
+  const [massagedAppliedPolicies, setMassagedAppliedPolicies] = React.useState(appliedPolicies);
+  const { enqueueSnackbar, closeSnackbar } = useSnackbar();
+
+  React.useEffect(() => {
+    const rootWorkloadApiForPods = seccompPolicies?.map((policy: any) => {
+      return FindRootWorkload(parseSeccompAnnotations(policy, 'pod', namespace), namespace);
+    });
+    rootWorkloadApiForPods &&
+      Promise.all(rootWorkloadApiForPods)
+        .catch((error: Error) => {
+          console.error(error);
+          return rootWorkloadApiForPods;
+        })
+        .then((rootWorkloads: any) => {
+          setMassagedSeccompPolicies((seccompPolicies: any) =>
+            seccompPolicies.map((policy: any, index: number) => {
+              policy.rootWorkload = rootWorkloads[index]?.kind;
+              policy.rootWorkloadName = rootWorkloads[index]?.name;
+              return { ...policy };
+            })
+          );
+        });
+  }, [seccompPolicies]);
+
+  function handleSave(value: any) {
+    updateSeccompPolicy(value)
+      .then(() => {
+        setIsEditorOpen(false);
+      })
+      .catch((err: Error) => {
+        setErrorMessage(err.message);
+      });
+  }
+
+  function handlePolicyDelete(policiesToDelete: any, resetSelectedPolicies: () => any) {
+    //it could be a delete request for an applied policy or a non-applied policy or a dangling policy so consider all
+    const policiesObjects = massagedSeccompPolicies
+      ?.concat(massagedAppliedPolicies)
+      ?.concat(danglingPolicies)
+      .filter((policy: any) => policiesToDelete.includes(policy?.metadata?.uid));
+    const policiesToDeleteName = policiesObjects
+      .map((policy: any) => policy?.metadata?.name)
+      .join(', ');
+    enqueueSnackbar(`Are you sure you want to delete policies ${policiesToDeleteName}`, {
```

Comment:
We have not used snackbars in other places in Headlamp for getting user confirmation, only for showing notifications (with optional actions to perform related to them).
The deleteButton shows a confirmation dialog should be an example of how to handle confirmation in a consistent way.